docs/standard/modern-web-apps-azure-architecture/common-web-application-architectures.md

```diff
@@ -145,49 +146,52 @@ In a Clean Architecture solution, each project has clear responsibilities. As su
 The Application Core holds the business model, which includes entities, services, and interfaces. These interfaces include abstractions for operations that will be performed using Infrastructure, such as data access, file system access, network calls, etc. Sometimes services or interfaces defined at this layer will need to work with non-entity types that have no dependencies on UI or Infrastructure. These can be defined as simple Data Transfer Objects (DTOs).
 
 > ### Application Core Types
-> -   Entities (business model classes that are persisted)
-> -   Interfaces
-> -   Services
-> -   DTOs
+>
```

Comment:
any special reason to add these lists inside quotation marks?

Reply:
No, someone converted my Word doc to Markdown and did that.